### PR TITLE
Create 2021-05-26-changes_to_the_nodejs_buildpack.md

### PR DESCRIPTION
**Pull request by mheadd**
Blog post with details about upcoming changes to the Node.js buildpack.

## Changes proposed in this pull request:
- Blog post to provide details of an upcoming change to the Node.js buildpack.

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/mheadd-patch-2/2021/05/26/changes_to_the_nodejs_buildpack/)

## Security Considerations
None

### CODE REVIEW

File: _posts/2021-05-26-changes_to_the_nodejs_buildpack.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: "Changes to the Node.js Buildpack"
+date: May 26, 2021
+excerpt: Certain versions of the Node.js runtime will be removed in upcoming releases of the Cloud Foundry Node.js buildpack.
+---
+
+The first release of the [Cloud Foundry Node.js buildpack](https://docs.cloudfoundry.org/buildpacks/node/node-tips.html#buildpack) after June 21, 2021 will no longer include Node versions 15.x.x. These Node.js versions will no longer be supported [upstream](https://github.com/nodejs/Release). 
+
+As this change may impact some cloud.gov users, application ownners are encouraged to migrate their Node.js apps to supported versions of Node.js before that time.
```
Comment (apburnes) [inline _posts/2021-05-26-changes_to_the_nodejs_buildpack.md]:
```suggestion
As this change may impact some cloud.gov users, application ownners are encouraged to migrate their Node.js apps to supported versions of Node.js before that time. _*Note: If no version is specified in your app's `package.json` file, the current default version resolves to Node v10.x._
```

Comment (apburnes) [reply _posts/2021-05-26-changes_to_the_nodejs_buildpack.md]: Not sure about this... Also surprised that the default is still 10.x

**Review by apburnes — DISMISSED** [review]
Maybe add info about the default version if folks are not specifying this for their apps. 🤷

File: _posts/2021-05-26-changes_to_the_nodejs_buildpack.md (hunk 1 above)
Comment (mheadd) [reply _posts/2021-05-26-changes_to_the_nodejs_buildpack.md]: Double checked. https://github.com/cloudfoundry/nodejs-buildpack/releases

**Review by apburnes — APPROVED** [review]

**Pull request merged**